Makefile

```diff
@@ -25,3 +25,12 @@ edge_cross_build:
 edgecontroller:
 	cd cloud/edgecontroller && $(MAKE)
 
+IMAGE_TAG ?= $(shell git describe --tags)
+
+.PHONY: cloudimage
+cloudimage:
+	docker build -t kubeedge/edgecontroller:${IMAGE_TAG} -f build/cloud/Dockerfile .
+
+.PHONY: certgenimage
```

Comment:
Do we really need the `make certgenimage` command? When do we need this command?

Reply:
The `certgen` is an image to generate certs. Because the script inside the image depends on `pr` command which may behaves weird in different linux dirstributions. To avoid this image comes.

It just an option not an necessary. I can remove this make rule.